pom.xml

```diff
@@ -535,6 +535,58 @@
           <artifactId>spring-boot-starter-logging</artifactId>
         </exclusion>
       </exclusions>
+      </dependency>
+    <dependency>
```

Comment:
hrm... can we pull in parquet without pulling in all of hadoop? i.e., minimize the dependencies we drag in...

Reply:
Yeah agreed, I initially tried to do it without hadoop using `parquet-mr`, but it was not available in any of the mirrors, and I kept getting `Missing artifact org.apache.parquet:parquet-mr:jar:1.12.3`.

So I had to fall back to hadoop

Reply:
what's the size of the fatjar before and after? i.e., how much are you pulling in?

maybe try suppressing the transitive inclusion of jars with exclude tags?

Reply:
master: 179MB
this branch: 265 MB (after putting in exclude tags)

Reply:
yikes! that's a lot of bloat for a relatively small feature...

can you poke around to see how we might make more lean?

Reply:
updated dependencies, it's around 220 MB now